EmployeeList/Services/EmployeesAPIService.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  EmployeeListService.swift
+//  EmployeeList
+//
+//  Created by Wang Siman on 1/29/21.
+//  Copyright © 2021 None. All rights reserved.
+//
+
+import Foundation
+import Combine
+
+final class EmployeesAPIService: APIServiceType {
+
+    internal let baseURL: String
```

Comment:
Shouldn't this be a URL, not a String?

Reply:
APIService protocol defines baseURL of String type. There is no specific reason I use String instead of URL... Now I know a better practice would be declaring a possibly url variable as URL.